### PR TITLE
Update homework 1

### PR DESCRIPTION
**Pull request by jlperona**
If you want to separate out the released files versus the future versions, you can still accept this PR and I can go back into the version history to dig out the file before I made changes

### CODE REVIEW

File: hw1/hw1.rst [hunk 1]
```diff
@@ -32,42 +32,46 @@ If you have any feedback about errors, big or small, please let me know!
...
 This assignment must be done alone.
 
-Step 1: gem5 Tutorial, Part I
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Step 1: Compile gem5
+~~~~~~~~~~~~~~~~~~~~
 
-Go through the Introduction and Part I of the `gem5 tutorial`__.
-You will go through the following topics:
+Go through the Introduction and Building gem5 pages of the `gem5 tutorial`__.
```
Comment (powerjg) [inline hw1/hw1.rst]: We can do this now, in a later change, or I can do it next time I teach this, but I'd like rename tutorial->book everywhere.

**Review by powerjg — COMMENTED** [review]
Thanks for cleaning all of this up. Let me know how you want to deal with the issue raised below. I'm fine with merging this as-is, if you think that's easiest.

File: hw1/hw1.rst (hunk 1 above)
Comment (jlperona) [reply hw1/hw1.rst]: I'll do it as part of this PR.

Comment (jlperona) [reply hw1/hw1.rst]: Resolved.

**Comment by powerjg** [comment]
Thanks!

**Pull request merged**